package.json

```diff
@@ -12,10 +12,10 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
+    "build": "DISABLE_ESLINT_PLUGIN=true react-scripts build",
+    "test": "DISABLE_ESLINT_PLUGIN=true react-scripts test",
+    "eject": "DISABLE_ESLINT_PLUGIN=true react-scripts eject"
```

Comment:
- [ ] Please remove the  ```DISABLE_ESLINT_PLUGIN=true``` from the scripts this is causing an error, when I remove it  and run `npm start` your app works properly, see the video below

https://user-images.githubusercontent.com/69638013/207633824-62eab39b-b95b-45cb-a629-87d75475d74f.mp4